src/settings.js

```diff
@@ -339,6 +339,13 @@ var EXCEPTION_DEBUG = 0;
 // [link]
 var DEMANGLE_SUPPORT = 0;
 
+// If 1, expose a formatException function that can be used to format a top
+// level C++ exception. If the thrown exception was
+// throw new std::runtime_error("a big problem occurred")
+// It returns a string like: "Cpp Exception std::runtime_error: A big problem occured."
+// [link]
+var FORMAT_EXCEPTION_SUPPORT = 0;
```

Comment:
Can we just avoid adding this new setting completely, and just ask folks to do `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=$formatException` on the command line?   

(I'm trying to avoid the never ending list of possible settings).

Reply:
(at least we can land it that way initially and add a specific setting later if it proves really useful).